web/src/components/dashboard/SidebarCardsList.tsx

```diff
@@ -36,14 +37,22 @@ export const SidebarCardsList = (props: Props): ReactElement => {
     }
   };
 
-  const showEmptyState = (): boolean => {
+  const showEmptyOpportunitiesMsg = (): boolean => {
     return (
       props.displayCertifications &&
       props.displayFundings &&
       props.certifications.length + props.fundings.length === 0
     );
   };
 
+  const showCompleteRequiredTasksMsg = (): boolean => {
+    return (
+      !props.displayCertifications &&
+      !props.displayFundings &&
```

Comment:
Pebble, based on your logic here, I think the renderComponent should have certifications and fundings. I think we render the component with the data already available but choose to render the cards when certain criteria is met.